cocos/ui/UIAbstractCheckButton.cpp

```diff
@@ -53,6 +54,11 @@ _frontCrossDisabledTexType(TextureResType::LOCAL),
 _zoomScale(0.1f),
 _backgroundTextureScaleX(1.0),
 _backgroundTextureScaleY(1.0),
+_backGroundFileName(""),
```

Comment:
Because lua extension for editor set these resource name by api call, it not set in editor. So we should save these name in ui component and when editor export project, get these file name from ui components.